src/server/onboarding-router/schema.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+
+export const ZOnboardMutationSchema = z.object({
+  user: z.object({
+    firstName: z.string(),
+    lastName: z.string(),
+    companyName: z.string(),
+    title: z.string(),
+  }),
+
+  company: z.object({
```

Comment:
I think that `companyName` should go here. Also I would keep them in similar formats and names as we save on database. So ideally, `org` table will be renamed to `company` and company columns are

* name
* incorporationType
* incorporationDate
* incorporationCountry
* incorporationState
* streetAddress
* city
* state
* zipcode